pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2868,6 +2882,46 @@
     ==
   ::
       %~2023.4.11
+    =,  server-state.old
+    =/  new-channel-state
+      :_  duct-to-key.channel-state
+      %-  ~(run by session.channel-state)
+      |=  c=channel-3
+      =/  new-events
+        =|  events=(qeu [id=@ud request-id=@ud =channel-event])
+        =/  l  ~(tap in ~(key by subscriptions.c))
+        |-
+        ?~  l  events
+        %=  $
+          l          t.l
+          next-id.c  +(next-id.c)
+          events     (~(put to events) [next-id.c i.l %kick ~])
+        ==
+      :*  mode.c
+          state.c
+          next-id.c
+          last-ack.c
+          new-events
+          ~
+          ~
+          heartbeat.c
+      ==
+    %=  $
+      old  ^-  axle
+           :*  %~2023.4.19
+               bindings
+               cache
+               cors-registry
+               connections
+               authentication-state
+               new-channel-state
+               domains
+               http-config
+               ports
+               outgoing-duct
+               0
+    ==     ==
```

Comment:
I'd put in a little comment explaining why we're kicking all the subscriptions here. Also, kindly consider `%=` instead of `:*`. Something along the lines of:

```suggestion
    =;  new-channel-sessions
      %=  $
        -.old                            %~2023.4.19
        session.channel-state.old        new-channel-sessions
        outgoing-duct.channel-state.old  [outgoing-duct.channel-state.old 0]
      ==
    %-  ~(run by session.channel-state.old)
    |=  c=channel-3
    =;  new-events
      c(events new-events, unacked ~, subscriptions ~)
    =|  events=(qeu [id=@ud request-id=@ud =channel-event])
    =/  l  ~(tap in ~(key by subscriptions.c))
    |-
    ?~  l  events
    %=  $
      l          t.l
      next-id.c  +(next-id.c)
      events     (~(put to events) [next-id.c i.l %kick ~])
    ==
```

Maybe there's a way to make this cleaner, but I always find explicitly doing the changes only is more legible than reconstructing the whole state.

Reply:
Done.